pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -798,7 +798,6 @@ protected BrokerResponse handleRequest(long requestId, String query, @Nullable S
         _brokerMetrics.addMeteredTableValue(rawTableName, BrokerMeter.BROKER_RESPONSES_WITH_NUM_GROUPS_LIMIT_REACHED,
             1);
       }
-      brokerResponse.setPartialResult(isPartialResult(brokerResponse));
```

Comment:
(minor) Also remove `isPartialResult()` method

Reply:
removed in 2d9649147559782e40fab44542dc93c0bb999249